### ISSUE TITLE
Triggering searching from a gene from html/javascript

### ISSUE THREAD

**Issue by davcraig75** [chanzuckerberg/cellxgene#2137]
We are using CellXGene as part of a larger project, where people type a gene and get data from a variety of resources - and one is CellXGene via a gene they have already searched on.

Is there a way via the HTML request or javascript to trigger a gene to be populated? 

 I've tried using javascript to trigger the button - while usually this would do it, it doesn't here. 
```
jQuery(".bp3-input-group .bp3-input").val("BRAF")
jQuery(".bp3-button.bp3-intent-primary".click()
jQuery(".bp3-input-group .bp3-input").simulate("key-sequence", {sequence: "APOE", delay: 10});
```

Alternatively, if a gene would appear via RESTful URL requests... 
```
/api/v0.2/data/var?var:name_0=APOE
```

**Comment by colinmegill** [comment]
@davcraig75 thanks a ton for filing this issue.

The click doesn't work because of Blueprint's Autosuggest mechanism. This is being replaced as we move to gene sets, so, we should take the autosuggest issue into account for #1861.

To clarify, do you mean you'd like to drive the client side via URL? 

cc @ambrosejcarr @bkmartinjr

**Comment by davcraig75** [comment]
Correct.  I am not sure if its a feature request or a question.  Is there a way to request a URL such that the gene loads in the initial view? I was looking on the REST API docs.  I would guess there is a way to control default layouts somehow with a post or something, so I thought I would ask.

**Comment by colinmegill** [comment]
With `gene sets`, which are merged to `main` but not yet deployed or in an official pypi release, you can drive multiple sets each with multiple genes from a CSV on load. Whatever is in the CSV will show up in a collapsable list in the right sidebar.

There is no ability to drive the app via URL or API at present.

**Comment by keithchev** [comment]
In case a specific example helps, what I think we would need in order to create links between OpenCell and cellxgene instances is for something like this to work:

`https://tabula-sapiens-cellxgene.ds.czbiohub.org/endothelial?genes=atl1,atl2,atl3` 

The idea being that this would open the cellxgene instance at `/endothelial` with the genes ATL1, ATL2, and ATL3 pre-added in the right-hand panel (or, in the future, with a gene set consisting of those genes already defined).

**Comment by ambrosejcarr** [comment]
The concrete case is really helpful. Thanks for posting it!

**Comment by brianherb** [comment]
[Nemo Analytics](https://nemoanalytics.org/) would like to also link to cellxgene in a gene specific manner. Our site displays many of the same BICCN datasets that cellxgene is hosting and would like to give our users the ability to switch back and forth between Nemo and cellxgene. For most of our data views, the user has selected a gene to view in the dataset, and we would like to provide a link to that same gene view in cellxgene. Thank you!
